src/ui/log_tab.rs

```diff
@@ -538,6 +546,20 @@ impl Component for LogTab<'_> {
 
                     self.describe_after_new = key.code == KeyCode::Char('N');
                 }
+                KeyCode::Char('s') => {
+                    self.popup = ConfirmDialogState::new(
+                        SQUASH_POPUP_ID,
+                        Span::styled(" Squash ", Style::new().bold().cyan()),
+                        Text::from(vec![
+                            Line::from("Are you sure you want to squash into this change?"),
```

Comment:
```suggestion
                            Line::from("Are you sure you want to squash @ into this change?"),
```

Updating for clarity where the changes are coming from